assets/sidecar/developer-hub-app-config-patches.yaml

```diff
@@ -0,0 +1,113 @@
+---
+apiVersion: v1
+data:
+  RHDH_TOKEN: base64-encoded-token-generated-from-openssl-or-similar-program
+kind: Secret
+metadata:
+  name: ai-rh-developer-hub-env
+  namespace: ai-rhdh
+type: Opaque
+
+---
+apiVersion: v1
+data:
+  app-config.extra.yaml: |-
+    auth:
+      environment: production
+      providers:
+        github:
+          production:
+            clientId: ${GITHUB__APP__CLIENT__ID}
+            clientSecret: ${GITHUB__APP__CLIENT__SECRET}
+    # start of delta from ai-rhdh-installer
```

Comment:
start of the changes I patch in to this configmap created by the ai-rhdh-installer ... I left the other parts for context